certify-pgdb-identity-cp1.properties

```diff
@@ -38,159 +45,112 @@ mosip.certify.identifier=${mosip.certify.domain.url}
+                            'locationOfLegalReserve': {'display': {'name': 'Location of Legal Reserve', 'locale': 'en'}}, \
+                            'registeredLegalReserveArea': {'display': {'name': 'Registered Legal Reserve Area', 'locale': 'en'}}, \
+                            'georeferencedLegalReserveArea': {'display': {'name': 'Georeferenced Legal Reserve Area', 'locale': 'en'}}, \
+                            'approvedLegalReserveAreaNotRegistered': {'display': {'name': 'Approved Legal Reserve Area Not Registered', 'locale': 'en'}}, \
+                            'proposedLegalReserveArea': {'display': {'name': 'Proposed Legal Reserve Area', 'locale': 'en'}}, \
+                            'totalLegalReserveDeclared': {'display': {'name': 'Total Legal Reserve Declared', 'locale': 'en'}}, \
+                            'permanentPreservationAreasAPP': {'display': {'name': 'Permanent Preservation Areas APP', 'locale': 'en'}}, \
+                            'appInConsolidatedRuralArea': {'display': {'name': 'APP in Consolidated Rural Area', 'locale': 'en'}}, \
+                            'appInNativeVegetationRemnantArea': {'display': {'name': 'APP in Native Vegetation Remnant Area', 'locale': 'en'}}, \
+                            'restrictedUseAreas': {'display': {'name': 'Restricted Use Areas', 'locale': 'en'}}, \
+                            'legalReserveDeficitExcess': {'display': {'name': 'Legal Reserve Deficit Excess', 'locale': 'en'}}, \
+                            'legalReserveAreaToRecompose': {'display': {'name': 'Legal Reserve Area to Recompose', 'locale': 'en'}}, \
+                            'permanentPreservationAreasToRecompose': {'display': {'name': 'Permanent Preservation Areas to Recompose', 'locale': 'en'}}, \
+                            'restrictedUseAreaToRecompose': {'display': {'name': 'Restricted Use Area to Recompose', 'locale': 'en'}}, \
+                            'embargoedAreaDescription': {'display': {'name': 'Embargoed Area Description', 'locale': 'en'}}, \
+                            'embargoedAreaProcessingDate': {'display': {'name': 'Embargoed Area Processing Date', 'locale': 'en'}}, \
+                            'embargoedAreaOverlap': {'display': {'name': 'Embargoed Area Overlap', 'locale': 'en'}}, \
+                            'embargoedAreaOverlapPercentage': {'display': {'name': 'Embargoed Area Overlap Percentage', 'locale': 'en'}}, \
+                            'conservationUnitDescription': {'display': {'name': 'Conservation Unit Description', 'locale': 'en'}}, \
+                            'conservationUnitProcessingDate': {'display': {'name': 'Conservation Unit Processing Date', 'locale': 'en'}}, \
+                            'conservationUnitOverlapArea': {'display': {'name': 'Conservation Unit Overlap Area', 'locale': 'en'}}, \
+                            'conservationUnitOverlapPercentage': {'display': {'name': 'Conservation Unit Overlap Percentage', 'locale': 'en'}}}\
```

Comment:
All these display are all english, portuguese entries are missing

Reply:
Can't resolve this due to hitting character limit.

Reply:
I'll skip English and have Portuguese as discussed.